### PR TITLE
Install guide review

### PR DESCRIPTION
**Pull request by jcayouette**
https://github.com/SUSE/spacewalk/issues/8363

- [X] #9041
- [ ] #9044 NEED INFO
- [ ] #9045 NEED INFO - I think copying activation keys is required for data parity?

- [X] intro (see // REMARKS)
- [X] gen req - added buttons for SCC UI
- [X] hard req - removed text from xrefs
- [X] net req - Updated links to SLES 15, removed text from xrefs
- [X] sup clients

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -84,7 +84,7 @@ clean: ## Remove build artifacts from output directory (Antora and PDF)
 
 # To build for suma-webui or uyuni you need to comment out the correct name/title in the antora.yml file. (TODO remove this manual method.)
 .PHONY: antora-suma
-antora-suma: clean pdf-all-suma ## Build the SUMA Antora static site (See README for more information)
+antora-suma: clean ##pdf-all-suma ## Build the SUMA Antora static site (See README for more information)
```
Comment (keichwa) [inline Makefile]:
```suggestion
antora-suma: clean pdf-all-suma ## Build the SUMA Antora static site (See README for more information)
```
committed by accident?

File: modules/installation/pages/general-requirements.adoc [hunk 2]
```diff
@@ -25,19 +25,18 @@ It is also not possible to use {slsa} for SAP as a base for {productname} for a
...
 == Obtaining your SUSE Customer Center Credentials
 
-You will need to create an account with {scc} before you install {sls} and {productname}.
-To obtain your {scc} credentials:
+Create an account with {scc} before installation of {sls} and {productname}.
 
 [[creating.scc.account.mgr]]
 .Procedure: Obtaining Your SCC Organization Credentials
 . Navigate to https://scc.suse.com/login in your Web browser.
 . Log in to your SCC account, or follow the prompts to create a new account.
```
Comment (keichwa) [inline modules/installation/pages/general-requirements.adoc]: In the following, these are not buttons but other GUI elements.  Thus I'd rather vote to use [guimenu] (or menu: if appropriate).

File: modules/installation/pages/install-intro.adoc [hunk 3]
```diff
@@ -21,4 +22,6 @@ This is the default method of installation.
 
 endif::[]
 
+
+// REMARK this para needs an update. We provide more than just jeos in this book.
```
Comment (keichwa) [inline modules/installation/pages/install-intro.adoc]: The rest is in the "if" above.  And then we must add a statement about proxy--just do it ;)

File: modules/installation/pages/general-requirements.adoc [hunk 4]
```diff
@@ -1,12 +1,12 @@
 [[installation-general-requirements]]
 = General Requirements
 
-
-Before you begin your installation, check that your environment meets these requirements:
+// REMARK this list needs to be cleaned up
+Before you begin installation, ensure that your environment meets the following requirements:
```
Comment (Loquacity) [inline modules/installation/pages/general-requirements.adoc]:
```suggestion
Before you begin installation, ensure that your environment meets these requirements:
```

Directional language.

File: modules/installation/pages/general-requirements.adoc [hunk 5]
```diff
@@ -25,19 +25,18 @@ It is also not possible to use {slsa} for SAP as a base for {productname} for a
 [[install.scc-register]]
 == Obtaining your SUSE Customer Center Credentials
 
-You will need to create an account with {scc} before you install {sls} and {productname}.
-To obtain your {scc} credentials:
+Create an account with {scc} before installation of {sls} and {productname}.
```
Comment (Loquacity) [inline modules/installation/pages/general-requirements.adoc]:
```suggestion
Create an account with {scc} before installation of {sls} and {productname}.
```

Why did you make this passive? The original worked better.

File: modules/installation/pages/install-intro.adoc [hunk 6]
```diff
@@ -1,6 +1,7 @@
 [[installation-intro]]
 = Introduction
 
+// REMARK we should update the intro. This is a marketing block for the product. It is not related to installation
```
Comment (Loquacity) [inline modules/installation/pages/install-intro.adoc]: Agreed. Just do it.

File: modules/installation/pages/network-requirements.adoc [hunk 7]
```diff
@@ -1,31 +1,26 @@
...
-
-
-
 This section details the networking and port requirements for {productname}.
 
 Fully Qualified Domain Name (FQDN)::
 The {productname} server must resolve its FQDN correctly or cookies will not work properly on the WebUI.
 +
-For more information about configuring the hostname and DNS, see https://www.suse.com/documentation/sles-12/book_sle_admin/data/sec_basicnet_yast.html#sec_basicnet_yast_change_host[SUSE Linux Enterprise Server Documentation - Configuring Host Name and DNS]
+For more information about configuring the hostname and DNS, see link:https://www.suse.com/documentation/sles-15/book_sle_admin/data/sec_network_yast.html#sec_network_yast_change_host[SUSE Linux Enterprise Server Documentation - Configuring Host Name and DNS]
```
Comment (Loquacity) [inline modules/installation/pages/network-requirements.adoc]:
```suggestion
For more information about configuring the hostname and DNS, see link:https://www.suse.com/documentation/sles-15/book_sle_admin/data/sec_network_yast.html#sec_network_yast_change_host
```

File: modules/installation/pages/network-requirements.adoc [hunk 8]
```diff
@@ -1,31 +1,26 @@
...
 
 Hostname and IP Address::
 To ensure that the {productname} domain name can be resolved by its clients, both server and client machines must be connected to a working DNS server.
 +
-For more information about setting up a DNS server, see https://www.suse.com/documentation/sles-12/book_sle_admin/data/cha_dns.html[SUSE Linux Enterprise Server Documentation - The Domain Name System]
+For more information about setting up a DNS server, see link:https://www.suse.com/documentation/sles-15/book_sle_admin/data/cha_dns.html[SUSE Linux Enterprise Server Documentation - The Domain Name System]
```
Comment (Loquacity) [inline modules/installation/pages/network-requirements.adoc]:
```suggestion
For more information about setting up a DNS server, see link:https://www.suse.com/documentation/sles-15/book_sle_admin/data/cha_dns.html
```

File: modules/installation/pages/network-requirements.adoc [hunk 9]
```diff
@@ -1,31 +1,26 @@
...
 
 Using a Proxy When Installing from {sle} Media::
 If you are on an internal network and do not have access to {scc}, you can set up and use a proxy during installation.
 +
-For more information about configuring a proxy for access to {scc} during a {sle} installation, see https://www.suse.com/documentation/sled-12/singlehtml/book_sle_deployment/book_sle_deployment.html#sec.i.yast2.start.parameters.proxy[SUSE Linux Enterprise Server Documentation -  Using a Proxy During Installation]
+For more information about configuring a proxy for access to {scc} during a {sle} installation, see link:https://www.suse.com/documentation/sles-15/book_sle_deployment/data/sec_boot_parameters_advanced.html#sec_boot_parameters_advanced_proxy[SUSE Linux Enterprise Server Documentation -  Using a Proxy During Installation]
```
Comment (Loquacity) [inline modules/installation/pages/network-requirements.adoc]:
```suggestion
For more information about configuring a proxy for access to {scc} during a {sle} installation, see link:https://www.suse.com/documentation/sles-15/book_sle_deployment/data/sec_boot_parameters_advanced.html#sec_boot_parameters_advanced_proxy
```

File: modules/installation/pages/network-requirements.adoc [hunk 10]
```diff
@@ -1,31 +1,26 @@
...
 
 
 .Naming Your Server
-[IMPORTANT]
+[CAUTION]
```
Comment (Loquacity) [inline modules/installation/pages/network-requirements.adoc]:
```suggestion
[IMPORTANT]
```

We have Note, Important, and Warning.

File: Makefile (hunk 1 above)
Comment (jcayouette) [reply Makefile]: woops yes.

File: modules/installation/pages/network-requirements.adoc (hunk 10 above)
Comment (jcayouette) [reply modules/installation/pages/network-requirements.adoc]: We have all five admon types? 

Note, Tip, Important, Warning and Caution.
Caution is something that can introduce breaking changes?

File: modules/installation/pages/general-requirements.adoc (hunk 5 above)
Comment (jcayouette) [reply modules/installation/pages/general-requirements.adoc]: Woops must have been a little tired thanks!

File: modules/installation/pages/network-requirements.adoc (hunk 10 above)
Comment (jcayouette) [reply modules/installation/pages/network-requirements.adoc]: I agree that caution here was incorrect. However we probably need to define our use of admons in our styleguide.

**Comment by jcayouette** [comment]
Fixed the intro.

**Pull request merged**